### PR TITLE
Add count endpoint for availableControler + IT

### PR DESCRIPTION
**Pull request by badekk**
## Add count endpoint for availableControler + IT

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

- [ ] Unit test(s)
- [x] Integration test(s)
- [ ] Tested manually

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings
- [ ] I have added frontend code and checked it for mobile & desktop

### CODE REVIEW

File: src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountResponse.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Orders.Actions.AvailableDeliveriesActions.AvailableDeliveriesCount
+{
+    public class AvailableDeliveriesCountResponse
+    {
+        public int Count { get; set; } = 1;
```
Comment (h4570) [inline src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountResponse.cs:5]: =1 do wywalenia, bo to było do mockupu

File: src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountService.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+            if (currentUser.Warehouse == null)
+            {
+                return new AvailableDeliveriesCountResponse
+                {
+                    Count = 0,
+                };
+            }
+
+            var delivererWarehouse = await _appDbContext.Users
```
Comment (h4570) [inline src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountService.cs:44]: Czemu zmienną się nazywa warehouse, a obiekt to user. Nazwa do zmiany.

**Review by h4570 — APPROVED** [review]
Możesz poprawić w tym lub nastepnym

File: src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountService.cs (hunk 2 above)
Comment (badekk) [reply src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountService.cs:44]: Damn chyba miało być deliverer WITH warehouse xd

File: src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountResponse.cs (hunk 1 above)
Comment (badekk) [reply src/backend/Orders/Actions/AvailableDeliveriesActions/AvailableDeliveriesCount/AvailableDeliveriesCountResponse.cs:5]: gotcha

**Pull request merged**